pkg/util/scrubber/scrubber_test.go

```diff
@@ -83,3 +83,13 @@ func TestScrubLine(t *testing.T) {
 	res := scrubber.ScrubLine("https://foo:bar@example.com")
 	require.Equal(t, "https://foo:********@example.com", res)
 }
+
+func TestGitleaks(t *testing.T) {
+	scrubber := New()
+	scrubber.gitleaks([]byte(`hello
+Split the adta by line.  Note that this does not copy the original data, Split the adta by line.  Note that this does not copy the original data,
+dynatrace secret is here: dt0c01.ST2EY72KQINMH574WMNVI7YN.G3DFPBEJYMODIDAEX454M7YWBUVEFOWKPRVMWFASS64NFH52PX6BNDVFFM572RZM!
+age secret is AGE-SECRET-KEY-1QQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQ <-- there
+Split the adta by line.  Note that this does not copy the original data, Split the adta by line.  Note that this does not copy the original data,
+hi`))
+}
```

Comment:
I'm not sure what this test is actually testing - is it just that gitleaks doesn't raise an error? If so, we should make sure we verify the resulting returned val too.

Reply:
nit: `adta` -> `data` x4